11c-check_compress_data.sh

```diff
@@ -164,7 +171,7 @@ done
 echo ""
 echo "Compressing the outputs from 03a, 10 and 11"
 cd ${home_directory}
-tar -zcf results/AgeSmokGWAS_${study_name}.tgz results/03/age_prediction.pdf results/03/age_prediction_correlation.png results/03/age_prediction_stats.csv results/03/age_prediction_stats_corrsd.csv results/10 results/11 
+tar -zcf results/AgeSmokGWAS_${study_name}.tgz results/03/logs_a results/03/age_prediction.pdf results/03/age_prediction_correlation.png results/03/age_prediction_stats.csv results/03/age_prediction_stats_corrsd.csv results/03/smoking_prediction.pdf results/10 results/11 
```

Comment:
IS results/03/logs_a a file or folder? If a folder, will this work?

Reply:
Sorry, i miss the log.txt in the end. I've fixed it.